channels/cr.m3u

```diff
@@ -9,8 +9,12 @@ http://rtmp.info:8081/anexiontv/envivo/index.m3u8
 https://rtmp.info/anexiontv/envivo/playlist.m3u8
 #EXTINF:-1 tvg-id="AntenaSeisTV.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://lh3.googleusercontent.com/V-_fSNpzlQPfFuUG0wLkrR7-PQ3RDx3n9tE4zA_8Exm_OKVeU24Kt2HiKLYweJXItgY" group-title="",Antena Seis TV (720p) [Geo-blocked]
 http://inliveserver.com:1935/14510/14510/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal6.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="" group-title="",Canal 6 (720p)
+https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal6/smil:live.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="Canal8.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://image.multimedios.cr/sites/default/files/styles/social_share/public/multimedios1.jpg" group-title="",Canal 8 (720p)
 http://mdstrm.com/live-stream-playlist/5a7b1e63a8da282c34d65445.m3u8
+#EXTINF:-1 tvg-id="Canal11.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="https://es.wikipedia.org/wiki/Canal_11_(Costa_Rica)#/media/Archivo:Repretel_11_logo.png" group-title="General",Canal 11
+https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal11/smil:live.smil/playlist.m3u8
```

Comment:
```suggestion
https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal11/smil:live.smil/playlist.m3u8
#EXTINF:-1 tvg-id="Canal2.cr" tvg-country="CR" tvg-language="Spanish" tvg-logo="" group-title="General",Repretel CDR-2 Central de Radios
https://d3ou4v1ntyo64z.cloudfront.net/repretel/canal2/smil:live.smil/playlist.m3u8
```

Reply:
It's a radio, hence the reason why I did not add it.